### PR TITLE
Spruce up Results Page

### PR DESCRIPTION
**Pull request by xiphirx**
The screenshot test result page looked a little sad / bland before, this
makes it look better (tm). I also inlined the view hierarchy since the
previous button to view it didnt work, and it didnt make much sense not
to. I also plan on converting the XML into a collapsable hierarchy at
some point, but this is a quick start on a redesign of the page

image

### CODE REVIEW

File: plugin/src/py/android_screenshot_tests/pull_screenshots.py [hunk 1]
```diff
@@ -52,27 +52,29 @@ def generate_html(dir):
...
         html.write('<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/themes/smoothness/jquery-ui.css" />')
         html.write('<link rel="stylesheet" href="default.css"></head>')
         html.write('<body>')
 
-        html.write('<!-- begin results -->')
-
         for screenshot in sort_screenshots(root.iter('screenshot')):
             alternate = not alternate
+            canonical_name = screenshot.find('name').text
+            last_seperator = canonical_name.rindex('.') + 1
```
Comment (dimorinny) [inline plugin/src/py/android_screenshot_tests/pull_screenshots.py]: Hi @xiphirx, this code crashed, when '.' symbol not found on name.
When I can read about this restriction?

Comment (xiphirx) [reply plugin/src/py/android_screenshot_tests/pull_screenshots.py]: Interesting... does your test class have no parent package?

Comment (dimorinny) [reply plugin/src/py/android_screenshot_tests/pull_screenshots.py]: No, I have a simple test
```
package com.avito.android.design.app.banner_card

class BannerCardTest {

    @Rule
    @JvmField
    val test = BannerCardRule()

    @Test
    fun screenshot__long_message_text_action_and_image() {
        test.bindData(
                BannerCardViewModel(
                        messageText = LONG_MESSAGE,
                        actionText = ACTION_TEXT,
                        image = IMAGE
                )
        )

        waitForImageBeingLoaded()

        test.recordScreenshot("Long_message_text_action_and_image")
    }
}
```
`recordScreenshot` implementation:

```
open fun recordScreenshot(description: String) {
        Component.ui.waitForIdle()

        Screenshot.snap(view)
                .setName(testSuiteName + "__" + description)
                .record()
    }
```

Comment (dimorinny) [reply plugin/src/py/android_screenshot_tests/pull_screenshots.py]: As result I have this name: `<name>BannerCardComponent__Long_message_text_action_and_image</name>`

Comment (xiphirx) [reply plugin/src/py/android_screenshot_tests/pull_screenshots.py]: Ah! Thanks for the report, I'll push a fix soon

**Pull request merged**